counterfit/core/logger.py

```diff
@@ -1,7 +1,20 @@
+import pathlib
+
 import orjson
-from counterfit.core.logging import CFAttackLogger
 
-class DefaultLogger(CFAttackLogger):
+
+class CFLogger:
+    """Base class for all loggers.
+
+    Returns:
+        [type]: [description]
```

Comment:
type and description are missing.

Reply:
Fixed in 47f2645b891acc8a3c9e1e1884d2d2f9eef14a45